src/c++/perf_analyzer/genai-perf/docs/files.md

```diff
@@ -0,0 +1,130 @@
+modification, are permitted provided that the following conditions
+are met:
+ * Redistributions of source code must retain the above copyright
+   notice, this list of conditions and the following disclaimer.
+ * Redistributions in binary form must reproduce the above copyright
+   notice, this list of conditions and the following disclaimer in the
+   documentation and/or other materials provided with the distribution.
+ * Neither the name of NVIDIA CORPORATION nor the names of its
+   contributors may be used to endorse or promote products derived
+   from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Generated File Structures
+
+## Overview
+
+This document serves as a guide to understanding the structure and contents of
+the files generated  by GenAi-Perf.
+
+## Directory Structure
+
+After running GenAi-Perf, your file tree should contain the following:
+
+```
+genai-perf/
+├── artifacts/
+│   ├── data/
+│   └── images/
+```
+
+## File Types
+Within the artifacts and docs directories, several file types are generated,
+including .gzip, .csv, .json, .html, and .jpeg. Below is a detailed
+explanation of each file and its purpose.
+
+### Artifacts Directory
+
+#### Data Subdirectory
+
+The data subdirectory contains the raw and processed performance data files.
+
+##### GZIP Files
+
+- all_data.gzip: Aggregated performance data from all collected metrics.
+- input_tokens_vs_generated_tokens.gzip: This contains data on the number of
+input tokens versus the number of generated tokens for each request.
+- request_latency.gzip: This contains the latency for each request.
+- time_to_first_token.gzip: Thsi contains the time to first token for each request.
+- token_to_token_vs_output_position.gzip: This contains the time from one token
+generation to the next versus the position of the output token for each token.
+- ttft_vs_input_tokens.gzip: This contains the time to first token versus
+the number of input tokens for each request.
+
+##### JSON Files
+
+- llm_inputs.json: This contains the input prompts provided to the LLM during testing.
+- profile_export.json: This is provided by Perf Analyzer and contains the timestamps
```

Comment:
@matthewkotila Any suggestions for improving this description? I don't know if we have documentation somewhere to point to. It sounds like we might want to even provide a tutorial for working with profile_export.json, though that'd likely live better in PA even if GenAi-Perf is the current use case.

Reply:
There's unfortunately no documentation anywhere currently describing the profile export JSON. There is a schema file that technically defines it but it isn't really a tutorial. It would be good to add documentation to PA to describe this file. I think this short description is good enough for now.

Reply:
Gotcha, thanks for clarifying. That sounds good!